clients/go/internal/containerSuite/containerSuite.go

```diff
@@ -46,54 +48,70 @@ func (s zeebeWaitStrategy) WaitUntilReady(ctx context.Context, target wait.Strat
 		return err
 	}
 
-	client, err := zbc.NewClient(&zbc.ClientConfig{
+	zbClient, err := zbc.NewClient(&zbc.ClientConfig{
 		UsePlaintextConnection: true,
 		GatewayAddress:         fmt.Sprintf("%s:%d", host, mappedPort.Int()),
 	})
 	if err != nil {
 		return err
 	}
 
-	defer func() {
-		err := client.Close()
-		if err != nil {
-			fmt.Println("Failed to close ZB client: ", err)
-		}
-	}()
+	res, err := zbClient.NewTopologyCommand().Send()
```

Comment:
Did it not work with using the ready check?

Reply:
No, the client's connection would still fail for one or two seconds so it was still necessary retry with the client

Reply:
I guess there might be some slight delay between the partition being installed and the command API being ready. Maybe we should revise that.